pkg/tfshim/schema/schema.go

```diff
@@ -145,7 +145,11 @@ func (m SchemaMap) Len() int {
 }
 
 func (m SchemaMap) Get(key string) shim.Schema {
-	return m[key]
+	s, ok := m[key]
+	if !ok {
+		panic("Missing key: " + key)
+	}
```

Comment:
This mirrors the error message for other `Get` calls. See:
https://github.com/pulumi/pulumi-terraform-bridge/blob/dde6f7ec58ab7d00a3a26c3bba867e5be527ebf9/pf/internal/schemashim/object_pseudoresource.go#L109-L115